test/language/expressions/template-literal/invalid-unicode-escape-sequence-truncated.js

```diff
@@ -9,4 +9,26 @@ info: >
 negative: SyntaxError
 ---*/
 
+`\01`;
+
+`\1`;
+
+`\xg`;
+
+`\xAg`;
+
 `\u0`;
+
+`\u0g`;
+
+`\u00g`;
+
+`\u000g`;
+
+`\u{g`;
+
+`\u{0`;
+
+`\u{10FFFFF}`;
+
+`\u{10FFFFF}${'inner'}right`;
```

Comment:
these cases should be split into different files for each case, otherwise you might find false positives for the pre-existing code.